src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -164,10 +164,13 @@ public void execute(Boolean arg) {
                      highlighters_.remove(webLinkHighlighter());
                   }
             }});
+            if (fileType.isR())
+               highlighters_.add(issueHighlighter());
             if (fileType != null && (fileType.isMarkdown() || fileType.isRmd()))
                highlighters_.add(markdownLinkHighlighter());
+
             nextHighlightStart_ = 0;
-            timer_.schedule(700);
+            timer_.schedule(100);
```

Comment:
Can you provide context for this change? IIRC the motivation here is to avoid the highlighter running while the user is actively typing / changing things in the document; 100ms is too low a threshold and so you'd get the highlighter eagerly running even while the user types.

Reply:
reverted it now, it was mostly because previously the markers were eliminated when e.g. the highlight markers (matching parenthesis etc ...).

Reply:
... actually setting the initial one, that runs on start (in the `AceEditorBackgroundLinkHighlighter` constructor), or when the mode changes to 100, and the ones reacting to document change to 700.